code/obj/item/clothing/shoes.dm

```diff
@@ -544,3 +544,17 @@
 	name = "jester's shoes"
 	desc = "The shoes of a not-so-funny-clown."
 	icon_state = "jester"
+
+#if ASS_JAM
```

Comment:
This object definition doesnt need to be locked behind a define if it's adminspawn- it makes it more difficult to use it elsewhere.
what *should* be behind that define, is it's availability in-game from vendors or traders.

Reply:
Made available to geoff, formatted properly (i hope)

Reply:
OH NO WHY IS IT FAILING ARGH

Reply:
close - you created the datums (which don't need to be behind the define either, they're just there for info) - now you need to add it to geoff's commodity list. ( in /obj/npc/trader/exclown ) 

that addition to his commodity list should be behind the define, to work as you intend!

Reply:
Stupid byond. Fixed.